pydis_site/apps/events/views/page.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from django.conf import settings
+from django.core.handlers.wsgi import WSGIRequest
+from django.http import Http404, HttpResponse
+from django.template import Context, Template
+from django.views import View
+
+PAGES_PATH = Path(settings.BASE_DIR, "pydis_site", "apps", "events", "pages")
+
+
+class PageView(View):
+    """Handles event pages showing."""
+
+    def get(self, request: WSGIRequest, path: str) -> HttpResponse:
+        """Render event page rendering based on path."""
+        # We need to get rid from trailing slash when path have this
+        if path.endswith("/"):
+            path = path[:-1]
```

Comment:
You can get rid of this if you're using `path`'s captured values, mentioned above.